.github/workflows/pr-verifier.yaml

```diff
@@ -18,6 +18,6 @@ jobs:
     steps:
     - name: Verifier action
       id: verifier
-      uses: kubernetes-sigs/kubebuilder-release-tools@v0.4.3
+      uses: kubernetes-sigs/kubebuilder-release-tools@012269a88fa4c034a0acf1ba84c26b195c0dbab4
```

Comment:
This is only half of what needs to be done for this particular action, because https://github.com/kubernetes-sigs/kubebuilder-release-tools/blob/012269a88fa4c034a0acf1ba84c26b195c0dbab4/action.yml#L12 . Perhaps we should take more drastic action, as suggested in https://github.com/kubernetes-sigs/kubebuilder-release-tools?tab=readme-ov-file#pr-verification-github-action-deprecated . However, we seem to be using a workaround that is not limited to March 18, so not sure what to do when.